packages/contracts-bridge/contracts/test/utils/MockHome.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+import {Message} from "@nomad-xyz/contracts-core/contracts/libs/Message.sol";
+
+contract MockHome {
```

Comment:
could this be reused from the `contracts-core` ?
if not, could it be in `contracts-router` so it could be useful to other people implementing routers?